draft-bonaventure-mptcp-converters.mkd

```diff
@@ -238,7 +255,7 @@ Client                   Converter                    Server
                                  <--------------------
                                          SYN+ACK 
      <--------------------
-       SYN+ACK,MPC [ No MPC ]
+       SYN+ACK,MPC 
```

Comment:
[No MPC] indicates that the client can learn that the server does not support MPTCP. This should be kept in the figure.